entry_types/scrolled/package/src/frontend/navigation/ChapterLinkTooltip.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import classNames from 'classnames';
+import headerStyles from "./AppHeader.module.css";
+import styles from "./ChapterLinkTooltip.module.css";
+import ReactTooltip from "react-tooltip";
+
+export function ChapterLinkTooltip(props) {
+  return(
+    <ReactTooltip id={props.chapterLinkId}
+                  type='light'
+                  place='bottom'
+                  effect='solid'
+                  className={classNames(headerStyles.navigationTooltip,
+                                        styles.chapterLinkTooltip)}>
+      <div>
+        <h3 className={styles.tooltipHeadline}>Kapitel {props.chapterIndex}</h3>
+        <p>
+          {props.summary}
+        </p>
```

Comment:
Summary can include HTML. I had [this commit](https://github.com/tf/pageflow/commit/c0c724c2b475c836f864d541658833f59537466a) on my outline branch, which I removed now since it was the only conflict between our branches and was somewhat unrelated.